pkg/common/helper_test.go

```diff
@@ -0,0 +1,167 @@
+package common
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type IsFileTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type IsDirTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type FileExistTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type StringSliceToIntSliceTestSuite struct {
+	suite.Suite
+}
+
+type RetryUntilSuccessfulTestSuite struct {
+	suite.Suite
+}
+
+func (suite *IsFileTestSuite) SetupTest()  {
+	var err error
+
+	// create a temp directory
+	suite.tempDir, err = ioutil.TempDir("", "isfile-test")
+	suite.Require().NoError(err)
+}
+
+func (suite *IsFileTestSuite) TestIsFileFuncPositive() {
+	//create temp file
+	tempFile, err := ioutil.TempFile(suite.tempDir, "temp_file")
+	suite.Require().NoError(err)
+	suite.Assert().True(IsFile(tempFile.Name()))
+	defer os.Remove(tempFile.Name())
+}
+
+func (suite *IsFileTestSuite) TestIsFileFuncFileIsNotExist() {
+	//create temp file
+	tempFile, err := ioutil.TempFile(suite.tempDir, "temp_file")
+	suite.Require().NoError(err)
+	os.Remove(tempFile.Name())
+	suite.Assert().False(IsFile(tempFile.Name()))
```

Comment:
Create and then remove? Why? Just look for a non-existent file within the dir

Reply:
In this case I'm not agree with you. How I can get a guarantee that file doesn't exist in any of systems which use nuclio if I just not deleted this file?